### ISSUE TITLE
aws_backup_vault does not reapply resource if manually removed account

### ISSUE THREAD

**Issue by DMonCode** [terraform-providers/terraform-provider-aws#11840]
### Terraform Version
Terraform v0.12.18
+ provider.aws v2.47.0
+ provider.template v2.1.2

NOTE: Tried same thing with v0.12.20. Same result
### Affected Resource(s)
aws_backup_vault
### Terraform Configuration Files

```hcl
resource "aws_backup_vault" "vault" {
  name        = "${var.environment}-backup-vault"
  kms_key_arn = aws_kms_key.backup_keys.arn

  tags = merge(
  local.common_tags
  ,
    map(
      "Name", "${var.environment}-protrans-efs-backup-vault"
    )
  )
}

```

### Expected Behavior
If that backup vault and backup plans are manually removed from the account then TF should rebuild those resources.

### Actual Behavior

```hcl
Error: error reading Backup Vault (sandbox-backup-vault): AccessDeniedException:
        status code: 403, request id: 580e7358-35dd-4c72-b8b0-c0293bbc236b
```

### Steps to Reproduce

1. Manually remove sandbox-backup-vault from account
2. `terraform apply`

Error will occur

### Important Factoids
If I manually add a backup_vault with the expected name then aws_backup_vault.vault will work even without an import.
 
I can run `terraform destroy -target aws_backup_vault.vault` and it will destroy the resource and any rule set for it. 
After terraform destroys the resource I can us `terraform apply` and it will rebuild the resource.

I cannot `terraform destroy -target aws_backup_vault.vault` unless the vault was manually added with the same name or added by terraform.

**Comment by DrFaust92** [comment]
i was able to reproduce the issue, open PR #11845 to address this

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.54.0 of the Terraform AWS Provider, later this week. Thanks to @DrFaust92 for the implementation. 👍
